portal/src/components/DownloadAsset/DownloadAsset.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import * as downloadjs from "downloadjs";
```

Comment:
```suggestion
import downloadjs from "downloadjs";
```

Dette fikser typefeilen